### DISCUSSION TITLE
what is the meaning of  “importance="anti" (equivalent to importance=TRUE)”？

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#236] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/5112397/166622363-d8ae8bc9-4e32-4815-aaf5-c3118cfb1cad.png)

library(randomForestSRC)

set.seed(123)
rf1 <- rfsrc(Species ~ ., data = iris, importance = TRUE)
plot(rf1)

set.seed(123)
rf2 <- rfsrc(Species ~ ., data = iris, importance = "anti")
plot(rf2)

![image](https://user-images.githubusercontent.com/5112397/166622153-e5611d7b-4fcc-4f34-a1a1-55167acdf35d.png)

Just as you can see，It is not the same

**Comment by ishwaran** [comment, marked as answer]
You are observing Monte Carlo differences.  `importance = TRUE ` is equivalent to `importance = "anti"`

For further details about reproducibility of VIMP see https://github.com/kogalur/randomForestSRC/issues/67#issuecomment-1100196455
